docs/installation.rst

```diff
@@ -25,9 +25,13 @@ Distribution Package
 --------------------
 
 Some distributions might offer a ready-to-use ``borgbackup``
-package which can be installed with the package manager.  As |project_name| is
-still a young project, such a package might be not available for your system
-yet.
+package which can be installed with the package manager.
+
+Those packages may not be up to date with the latest Borg releases
+and problems with those packages should first be reported to the
+operating system package maintainers instead of upstream |project_name|.
+
```

Comment:
Well, in some cases this might make sense, in others it doesn't.
I've often seen bugs being reported to some linux distribution that should have been reported (and fixed) upstream. And not always maintainers forward the bug, so the issue ends up rotting in some distribution issue tracker.